pom.xml

```diff
@@ -429,6 +433,27 @@
             <groupId>com.coravy.hudson.plugins.github</groupId>
             <artifactId>github</artifactId>
             <version>1.28.0</version>
+            <exclusions>
+                <!-- Upper bound fix.
+                     TODO: Remove it once github plugin commit
+                     https://github.com/jenkinsci/github-plugin/commit/a9308f124eeb5e599b2564d6355c5dd5cdafee34
+                     is released -->
+                <exclusion>
+                    <groupId>org.jenkins-ci</groupId>
+                    <artifactId>symbol-annotation</artifactId>
+                </exclusion>
+                <!-- Upper bound fix: Remove it once token-micro plugin is fixed to remove dependency on asm-tree
+                     PR: https://github.com/jenkinsci/token-macro-plugin/pull/31
```

Comment:
https://github.com/jenkinsci/token-macro-plugin/commit/66ede22a9fefe9f3a459472a60f8230bc3c50851 now (unreleased)

Reply:
ack, added to comment